simpleheat.js

```diff
@@ -96,6 +96,12 @@ simpleheat.prototype = {
     },
 
     draw: function (minOpacity) {
+    	//return if not ready
+        if (!this._isReady()) {
+        	return this;
+        }
```

Comment:
I think we can also inline `_isReady` here since it's small and not used anywhere else. And remove the curly braces, making it a one-line if/return.